tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_interval.py

```diff
@@ -0,0 +1,70 @@
+import os
+import pytest
+
+from wazuh_testing import global_parameters
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.vulnerability_detector import callback_detect_vulnerability_scan_sleeping
+from wazuh_testing.tools.time import time_to_human_readable, unit_to_seconds
+
+
+# Marks
+pytestmark = pytest.mark.tier(level=0)
+
+# variables
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_interval.yaml')
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+interval_values = [1,2,5]
+interval_units  = ['s','m','h','d']
+
+parameters = []
+metadata = []
+ids = []
+
+for value in interval_values:
+  for unit in interval_units:
+    parameters.append({'INTERVAL': f'{value}{unit}'})
+    metadata.append({'interval': f'{value}{unit}'})
+    ids.append(f'{value}{unit}')
```

Comment:
Tabs size must be 4 spaces

Reply:
Of course https://github.com/wazuh/wazuh-qa/pull/686/commits/306de9ae9263f93ba8dd1525933cec0c326bd9b0